CONTRIBUTING.md

```diff
@@ -1,10 +1,42 @@
 
-# How to contribute
+# How you can contribute
+The BC Design System is for everyone, and needs a strong community behind it to make it successful. There are lots of ways to contribute – reporting bugs, editing documentation, participating in discussions, or even creating new components. Your contribution to it will ensure it helps meets user needs from across the BC government. 
```

Comment:
Being ruthless here so do ignore my edits if you think otherwise.

rephrase to:
"The BC Design System needs a strong community behind it to make it successful.

Reply:
"You can contribute by reporting bugs, editing documentation, participating in discussions, or even creating new components."